pppd: Don't use exponential timeout in discovery phase

This patch removes the exponential timeout increase between PADO or PADS
discovery attempts.

Signed-off-by: Jo-Philipp Wich <jo@mein.io>

--- a/pppd/plugins/rp-pppoe/discovery.c
+++ b/pppd/plugins/rp-pppoe/discovery.c
@@ -644,7 +644,9 @@ discovery(PPPoEConnection *conn)
 	conn->discoveryState = STATE_SENT_PADI;
 	waitForPADO(conn, timeout);
 
+#if 0
 	timeout *= 2;
+#endif
     } while (conn->discoveryState == STATE_SENT_PADI);
 
     timeout = conn->discoveryTimeout;
@@ -659,7 +661,9 @@ discovery(PPPoEConnection *conn)
 	sendPADR(conn);
 	conn->discoveryState = STATE_SENT_PADR;
 	waitForPADS(conn, timeout);
+#if 0
 	timeout *= 2;
+#endif
     } while (conn->discoveryState == STATE_SENT_PADR);
 
     if (!conn->seenMaxPayload) {
